common/scala/src/main/resources/application.conf

```diff
@@ -81,7 +81,9 @@ whisk {
             // A low value improves latency performance but it is important to not set it too low
             // as that will cause excessive busy-waiting.
             fetch-max-wait-ms = 20
-            metric-flush-interval-s = 60
+
+            // Used to control the window size of kafka metric reporting and cadence of consumer lag checking
+            metrics-sample-window-ms = 60000 // 60 seconds
```

Comment:
@sven-lange-last made me aware of the fact that this a really bad idea because of conflating two things. Will update.